src_connection_IAM.tf

```
@@ -0,0 +1,20 @@
+data "aws_iam_policy_document" "dw_ksr_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+resource "aws_iam_role" "dw_ksr_s3_readonly" {
+  name               = "dw_ksr_s3_readonly"
```

We really don't have a standard format for names, sadly. There's a mix of hyphens and under scores, but you don't need to state it's DataWorks, but I think the full name instead of ksr is better
```suggestion
  name               = "kick_start_s3_readonly"
```

---

can I put kickstart_s3_readonly, because this data is related to kickstart scheme launched by Govt., So it is just one word ?

---

yep, that would be fine

---

Changes has been made in the file the new role name is kickstart_s3_readonly